spec/rails_helper.rb

```diff
@@ -568,22 +551,6 @@ class SpecTimeoutError < StandardError
 
   config.before(:each, type: :system) do |example|
     if !system_tests_initialized
-      # Use a file system lock to get `selenium-manager` to download the `chromedriver` binary that is required for
-      # system tests to support running system tests in multiple processes. If we don't download the `chromedriver` binary
-      # before running system tests in multiple processes, each process will end up calling the `selenium-manager` binary
-      # to download the `chromedriver` binary at the same time but the problem is that the binary is being downloaded to
-      # the same location and this can interfere with the running tests in another process.
-      #
-      # The long term fix here is to get `selenium-manager` to download the `chromedriver` binary to a unique path for each
-      # process but the `--cache-path` option for `selenium-manager` is currently not supported in `selenium-webdriver`.
-      File.open("#{Rails.root}/tmp/chrome_driver_flock", File::RDWR | File::CREAT, 0644) do |file|
```

Comment:
We won’t need this anymore since the chrome driver will be installed by Debian at `/usr/bin/chromedriver`.

Reply:
We will still need this for people running system tests locally with `turbo_rspec`. They can still run into problems if their chrome browser is updated and selenium-manager needs to install the chrome driver for that particular browser version.

Reply:
Running one spec in isolation will take care of the problem in theory (the chrome driver will have been downloaded by selenium-manager then), but I guess it doesn’t hurt to keep it for now :slightly_smiling_face: